README.md

```diff
@@ -119,37 +117,39 @@ You can use KotlinSyft as a front-end or as a background service. The following
                     )
                 )
                 // your custom implementation to read a databatch from your data
-                val batchData = dataRepository.loadDataBatch(clientConfig.batchSize)
+                val batchData = dataRepository.loadDataBatch(
+		    (clientConfig.planArgs["batch_size"] ?: error("batch_size doesn't exist")).toInt()
+		)
                 //get Model weights and return if not set already
-                val modelParams = model.getParamArray() ?: return
+                val modelParams = model.paramArray ?: return
                 val paramIValue = IValue.listFrom(*modelParams)
                 // plan.execute runs a single gradient step and returns the output as PyTorch IValue
-                val output = plan.execute(
+                val output = plan?.execute(
                     batchData.first,
                     batchData.second,
                     batchIValue,
                     lr,paramIValue
                 )?.toTuple()
                 // The output is a tuple with outputs defined by the pysyft plan along with all the model params
                 output?.let { outputResult ->
-                    val paramSize = model.modelState!!.syftTensors.size
+                    val paramSize = model.stateTensorSize!!
                     // The model params are always appended at the end of the output tuple
                     val beginIndex = outputResult.size - paramSize
                     val updatedParams =
                             outputResult.slice(beginIndex until outputResult.size)
                     // update your model. You can perform any arbitrary computation and checkpoint creation with these model weights
-                    model.updateModel(updatedParams.map { it.toTensor() })
+                    model.updateModel(updatedParams)
                     // get the required loss, accuracy, etc values just like you do in Pytorch Android
                     val accuracy = outputResult[0].toTensor().dataAsFloatArray.last()
                 }
             }
             // Once training finishes generate the model diff
-            val diff = mnistJob.createDiff()
+            val diff = newJob.createDiff()
             // Report the diff to PyGrid and finish the cycle
-            mnistJob.report(diff)
+            newJob.report(diff)
         }
 
-        override fun onRejected() {
+        fun onRejected() {
```

Comment:
onRejected should be available as subscriber method. I think override should be there

Reply:
I removed this because the IDE complained about it. I can put it back if you prefer.

Reply:
I see because it's actually onRejected(timeout: String)

Reply:
Oh! I should have noticed that. Thanks for finding this. I just updated the signature.